utopia-remix/app/services/fgaClient.server.ts

```diff
@@ -46,7 +46,10 @@ export function createFgaClient() {
     return mockOpenFgaClient
   }
   return new WriteSafeOpenFgaClient({
-    apiScheme: ServerEnvironment.FGA_API_SCHEME,
+    apiScheme:
+      ServerEnvironment.environment === 'local' || ServerEnvironment.environment === 'test'
```

Comment:
no, this is not actually the the correct logic... It should be according to `fgaclientMode` if we want to decide it here